base/deprecated.jl

```diff
@@ -1540,4 +1540,15 @@ unsafe_wrap(::Type{String}, p::Cstring, len::Integer, own::Bool=false) =
 # Rename LibGit2.GitAnyObject to LibGit2.GitUnknownObject (part of #19839)
 eval(LibGit2, :(Base.@deprecate_binding GitAnyObject GitUnknownObject))
 
+# Deprecate one argument method in order to make chomp=true the new default  #19944
```

Comment:
maybe add a note here to put the default back (i.e. revert b8ab89a28211ee4c5d6d3f2c488c181cc61746cb) when these deprecations get removed?